apps/cookbook/src/app/showcase/showcase.component.ts

```diff
@@ -36,7 +42,10 @@ export class ShowcaseComponent implements OnDestroy {
     this.exampleComponentPopOutUrl = ['/', 'examples', exampleComponentUrlSegment];
     this.exampleComponentGitUrl = this.gitUrl + exampleComponentUrlSegment + '-example';
     this.exampleComponentName = this.replaceHyphens(exampleComponentUrlSegment);
-    this.isCTABoxShown = this.exampleComponentName !== 'colors';
+    this.areCTAlinkShown = this.exampleComponentName !== 'colors';
+    window.setTimeout(() => {
+      this.propertiesTable = document.getElementsByClassName('cookbook-properties')[0];
+    }, 1); // this queues the query to ensure DOM has been rendered
```

Comment:
This probably could/should be achieved through querying ContentChildren instead...

Reply:
Probably should, yes :)
But I haven't found any examples of querying a viewChild by class name (the tables are not added consistently.. which of course should be fixed, but I figured I had to pick my battles in this PR).
If you know of a good way to do it, I am all ears